hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -117,18 +117,22 @@ public final class HoodieMetadataConfig extends HoodieConfig {
       .withDocumentation("Controls how often the metadata table is compacted.");
 
   // Max number of seconds before compaction occurs
-  public static final ConfigProperty<String> INLINE_COMPACT_TIME_DELTA_SECONDS = ConfigProperty
-      .key(METADATA_PREFIX + ".compact.inline.max.delta.seconds")
+  public static final ConfigProperty<String> COMPACT_TIME_DELTA_SECONDS = ConfigProperty
+      .key(METADATA_PREFIX + ".compact.max.delta.seconds")
```

Comment:
For MDT, should the default compaction interval seconds be smaller, like 1 hour instead of 24 hours?